m/main_glider_data_processing_dt.m

```diff
@@ -233,6 +252,7 @@
   netcdf_l0_file = strfstruct(config.paths_local.netcdf_l0, deployment);
   netcdf_l1_file = strfstruct(config.paths_local.netcdf_l1, deployment);
   netcdf_l2_file = strfstruct(config.paths_local.netcdf_l2, deployment);
+  matlab_l2_file = strfstruct(config.paths_local.matlab_l2, deployment);
```

Comment:
I guess that you added a new output type.
It is ok, but please note that it will be removed during the merge.
You might recover it later by checking out the file from your local branch.